src/Extensions/Aspects/Microsoft.Omex.Extensions.Aspects.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(TargetFrameworksValue)</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="MethodBoundaryAspect.Fody" Version="2.0.118" />
```

Comment:
Should this be in dependencies.props?

Reply:
Yes, it should, it's still a draft I'll be changing it couple times

Reply:
Replaced with version variables